boudicca.base/enricher/src/main/kotlin/base/boudicca/enricher/EnricherConfiguration.kt

```diff
@@ -20,10 +23,6 @@ import org.springframework.scheduling.annotation.EnableScheduling
         )
     ),
 )
-@SpringBootApplication
-@EnableScheduling
-class EnricherApplication
-
-fun main(args: Array<String>) {
-    runApplication<EnricherApplication>(*args)
-}
+@AutoConfiguration
+@ComponentScan
```

Comment:
Should we really use component scan here? to me it seems this is obfuscating a lot of magic that happens under the hood.

When implementing an enricher for some kind of boudicca derivative service I would like to choose which enrichers I want to use and not magically pull in all of them, right?

Reply:
"When implementing an enricher for some kind of boudicca derivative service I would like to choose which enrichers I want to use and not magically pull in all of them, right?"

which is exactly why i made the split. the `ComponentScan` only detects the controller and the enricherservice itself, not actual implementations of enrichers, which now are part of the actual enricher, which is custom to boudicca.events and whoever wants to do their own implementation